data/helper/data_dictionary.csv

```diff
@@ -33,12 +35,13 @@ Text,genre_id,Refer to genre IS,,2017-07-04,LH,2017-07-04,LH
 Text,publication_place,place of first publication,,2017-07-04,LH,2017-07-04,LH
 Text,publication_date,date of first publication in original language,,2017-07-04,LH,2017-07-04,LH
 Text,publishing_house,name of publishing house,provide pinyin for Chinese publishing houses,2017-07-04,LH,2017-07-04,LH
-Text,first_chin_edition,year of first edition in China,,2017-07-04,LH,2017-07-04,LH
-Text,physical_description,please add information in two cases: 1) if you have any information about the physical condition of the object 2) for internet source add: 'Internet Media Type',,,,,
-Text,neibu,yes/no whether this was originally a neibu publication,,2017-07-04,LH,2017-07-04,LH
+Text,first_chin_edition,"year of first edition in China. This is a nice-to-have category, please do not invest too much time in finding out! We will keep it for the moment, but the contained information may be deleted or copied elsewhere.",,2017-07-04,LH,2017-07-04,LH
+Text,physical_description,please add information in two cases: 1) if you have any information about the physical condition of the object 2) for internet source add: 'Internet Media Type',,2017-10-12,OS,2017-10-12,OS
+Text,neibu,"yes/no whether this was originally a neibu publication, add „NULL“ for every publication starting from 1980",,2017-07-04,LH,2017-07-04,LH
```

Comment:
please avoid the german U+201E and U+201F characters ``„‟``, they technically don't exist in english. Most importantly, they often interfere with the parsing of the csv file. Use two U+0027 ``'`` inside a cell, or if nested quotes are necessary U+0022 ``"``

Reply:
#104